rholang/src/test/scala/coop/rchain/rholang/interpreter/RegistrySpec.scala

```diff
@@ -58,7 +55,9 @@ trait RegistryTester extends PersistentStoreTester {
   ): R =
     withTestSpace(errorLog) {
       case TestFixture(space, _) =>
-        val _                                        = errorLog.readAndClearErrorVector().runSyncUnsafe(1.second)
+        val _             = errorLog.readAndClearErrorVector().runSyncUnsafe(1.second)
```

Comment:
we keep adding runSyncUnsafe calls. At some moment we should group these in a task on which we only call runSyncUnsafe once. Maybe that moment is now, if you feel like it. No pressure. https://www.youtube.com/watch?v=JfnddMpzPsM